fast64_internal/oot/oot_scene_room.py

```diff
@@ -593,6 +310,9 @@ class OOTSceneHeaderProperty(bpy.types.PropertyGroup):
 	exitList : bpy.props.CollectionProperty(type = OOTExitProperty, name = "Exit List")
 
 	writeCutscene : bpy.props.BoolProperty(name = "Write Cutscene")
+	csWriteType : bpy.props.EnumProperty(name = "Cutscene Data Type", items = ootEnumCSWriteType, default = "Embedded")
+	csWriteCustom : bpy.props.StringProperty(name = "CS hdr var:", default = "")
+	csWriteObject : bpy.props.PointerProperty(name = "Cutscene Object", type = bpy.types.Object)
 	csEndFrame : bpy.props.IntProperty(name = "End Frame", min = 0, default = 100)
```

Comment:
Isn't a lot (all?) of the stuff here duplicate of what `oot_cutscene.py` contains?

Reply:
Yes, that is intentional. This is the deprecated "embedded" cutscene system, where the cutscene data is stored in the scene header itself. Removing this would break any cutscenes which anyone (me!) had already made with that system. However, please note that it's just these 8 properties which are duplicated. Everything within `OOTCSListProperty`, including all the GUI for editing this property, is shared between the "embedded" system and the new first-class cutscene object system.

Reply:
Could you make it obvious what is legacy code (kept for compatibility) by adding comments wherever needed

Reply:
Then if you can confirm the addon isn't broken with master merged (since you moved stuff around, but there hasn't been activity on the oot side recently so it should be fine) we can merge this